### PR TITLE
Fix: use of overmap camera takes away mob's ability to move

### PR DESCRIPTION
**Pull request by covertcorvid**
## About The Pull Request
Following the ship vibe check the mob's relaymove wasn't reset after use of overmap cameras. This includes both bridge consoles and any seegson ship viewscreen. After stopping the use of the camera, the mob would be unable to move and use of movement keys would cause runtime errors.

## Why It's Good For The Game
Movement is good

## Changelog
:cl:
fix: mobs can move normally after exiting overmap camera view
/:cl:

### CODE REVIEW

File: nsv13/code/modules/overmap/camera.dm [hunk 1]
```diff
@@ -60,6 +60,7 @@
 	if(eyeobj?.off_action)
 		qdel(eyeobj.off_action)
```
Comment (francinum) [inline nsv13/code/modules/overmap/camera.dm]: This doesn't look right to me...

File: nsv13/code/modules/overmap/camera.dm [hunk 2]
```diff
@@ -60,6 +60,7 @@
...
 	M.cancel_camera()
+	M.remote_control = null
```
Comment (Jalleo) [inline nsv13/code/modules/overmap/camera.dm]: This likely fixes the issue but can you explain to Fran why the code qdel's above?

**Comment by covertcorvid** [comment]
With regards to the implicit null, qdel waits 2 minutes before deleting to see if garbage collection takes care of it. https://github.com/Baystation12/Baystation12/wiki/Guide-to-Garbage-Collection
Not sure why it qdels, not my code, will try to analyze it more for you later

**Comment by covertcorvid** [comment]
@francinum @Jalleo Looking at this further, eyeobj itself wasn't being deleted, off_action was, so even if deletion was immediate, which it's not, relaymove's reference still wouldn't be null because eyeobj would still exist.
Since off_action maintains an internal reference to eyeobj, even without relaymove pointing to eyeobj, neither of these would ever be GC'd if you don't either free one of the references manually or qdel one of them.

**Pull request merged**